src-vue/src/components/PullRequestsSelector.vue

```diff
@@ -0,0 +1,186 @@
+            const openedCollapseNames = Object.values(e);
+            if (openedCollapseNames.includes('1') && this.pull_requests_launcher.length === 0) {
+                this.getPullRequests('LAUNCHER');
+            }
+            if (openedCollapseNames.includes('2') && this.pull_requests_mods.length === 0) {
+                this.getPullRequests('MODS');
+            }
+        },
+        async getPullRequests(pull_request_type: PullRequestType) {
+            await invoke<PullsApiResponseElement[]>("get_pull_requests_wrapper", { installType: pull_request_type })
+                .then((message) => {
+                    switch (pull_request_type) {
+                        case "MODS":
+                            this.$store.state.pull_requests_mods = message;
+                            break;
+
+                        case "LAUNCHER":
+                            this.$store.state.pull_requests_launcher = message;
+                            break;
+
+                        default:
+                            console.error("We should never end up here");
+                    }
+                })
+                .catch((error) => {
+                    ElNotification({
+                        title: 'Error',
+                        message: error,
+                        type: 'error',
+                        position: 'bottom-right'
+                    });
+                });
+        },
+        async installLauncherPR(pull_request: PullsApiResponseElement) {
+            // Send notification telling the user to wait for the process to finish
+            const notification = ElNotification({
+                title: `Installing launcher PR ${pull_request.number}`,
+                message: 'Please wait',
+                duration: 0,
+                type: 'info',
+                position: 'bottom-right'
+            });
+
+            await invoke("apply_launcher_pr", { pullRequest: pull_request, gameInstallPath: this.$store.state.game_path })
+                .then((message) => {
+                    console.log(message);
+                    // Show user notification if mod install completed.
+                    ElNotification({
+                        title: `Done`,
+                        message: `Installed ${pull_request.number}: "${pull_request.title}"`,
+                        type: 'success',
+                        position: 'bottom-right'
+                    });
+                })
+                .catch((error) => {
+                    ElNotification({
+                        title: 'Error',
+                        message: error,
+                        type: 'error',
+                        position: 'bottom-right'
+                    });
+                })
+                .finally(() => {
+                    // Clear old notification
+                    notification.close();
+                });
+        },
+        async installModsPR(pull_request: PullsApiResponseElement) {
```

Comment:
I think that for more clarity, we should create a store module dedicated to PR installation with this method.

Reply:
Done in 068d1e56ab428b4b94bfeab9943019a4f26db9e8